tensorflow/stream_executor/rocm/BUILD

```diff
@@ -274,6 +274,16 @@ cc_library(
     alwayslink = True,
 )
 
+cc_library(
+        name = "rocm_helpers",
+        srcs = ["rocm_helpers.cu.cc"],
```

Comment:
Nit: Be more specific in naming. Maybe `rocblas_gemm_batched_helper`.

Reply:
Nit: Add the ticket number to it and a TODO, so that when rocblas fixed their API issue, we remember to remove it.

Reply:
Depends on whether we want one file for all kernels in stream_executor/rocm (as I was assuming), or a separate file for each.

Reply:
I haven't seen any other kernel appears in this folder. I had the feeling that this is what TF is designed to avoid to as well (that all kernels goes under tf/core/kernels so that se is decoupled from it). Based on this, the fix should be temporary in nature and we'd try not to add more kernels in here. If we can agree on this, naming does not matter as much in this case.

Reply:
This is, I think, the third time I feel the need to add kernel code in stream_executor. So it's a recurrent problem and it'll probably arise again in future. And at least this fix is not going anywhere any time soon, based on rocBLAS feedback. 

TF is also (sort of) designed to separate platform-independent logic (into core/kernels), and platform-dependent logic (into stream_executor/rocm or stream_executor/cuda). When we need custom processing to plug a hole in rocBLAS, MIOpen, etc., the choice is between taking care of things inside stream_executor/rocm, or creating a mess in the public areas like core/kernels (and possibly even changing the stream_executor API to accommodate our quirks.) It seems cleaner to keep all band-aids out of the public areas.

Reply:
I see the point in your reasoning. On a second thought, I, too don't think there is a clean way to fix it. After all, we don't want to update every call site for blas functions.

Does this aims to be "temporary", in terms that we don't expect this to be upstreamed? My concerns are all based on the assumption that we'll upstream this.

Reply:
I expect it to be upstreamed. If "they" object to kernel code with stream_executor, we'll think of what to do at that point.

Reply:
Sounds good, thanks.